src/AzureIoTHub.Portal/Client/Shared/Appbar.razor

```diff
@@ -0,0 +1,49 @@
+﻿@using AzureIoTHub.Portal.Models.v10
+@using Microsoft.AspNetCore.Components.WebAssembly.Authentication
+
+@inject PortalSettings Portal
+@inject SignOutSessionStateManager SignOutManager
+@inject NavigationManager Navigation
+@inject ILayoutService LayoutService
+
+<MudIconButton Icon="@Icons.Material.Filled.Menu" Color="Color.Inherit" Edge="Edge.Start" OnClick="@LayoutService.ToggleDrawer" />
+<MudText Typo="Typo.h5">@Portal.PortalName</MudText>
+<MudSpacer />
+<AuthorizeView>
+    <Authorized>
+        <MudMenu AnchorOrigin="Origin.BottomRight" TransformOrigin="Origin.TopRight">
+            <ActivatorContent>
+                <MudAvatar Variant="Variant.Outlined">
```

Comment:
```suggestion
                <MudAvatar>
```

To remove the border on the avatar in order to get it simple: 
![image](https://user-images.githubusercontent.com/9513635/173995928-5c41abce-31ee-4923-95dd-aad00977606a.png)